symplyphysics/laws/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index.py

```diff
@@ -0,0 +1,57 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+)
+
+# Description
+## Newton's rings are annular interference maxima and minima that appear around the point of contact between
+## a convex lens and a plane—parallel plate when light passes through the lens and plate.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
+## beam reflected from the convex surface of the lens at the glass-air boundary and the beam reflected from the
+## plate at the air-glass boundary is equal to an entire wave "n", then these two waves hit the observation
+## point with the same faces and demonstrate interference.
+
+## Law is: r = sqrt(k * R * L / n), where
+## r - radius of Newton's ring,
+## k - order of interference,
+## R - radius of curvature of lens
+## L - wavelength in a vacuum,
+## n - refractive index of medium between lens and plate.
+
+radius = Symbol("radius", units.length)
+
+wavelength = Symbol("wavelength", units.length)
+double_order_interference = Symbol("double_order_interference", dimensionless)
+radius_curvature = Symbol("radius_curvature", units.length)
+refractive_index = Symbol("refractive_index", dimensionless)
```

Comment:
Mention the medium. It is easy to misunderstand it for refractive index of lens.